pages/memory-list/index.tsx

```diff
@@ -1,9 +1,14 @@
 import { GetServerSideProps, InferGetServerSidePropsType } from 'next'
-import { useCallback, useEffect, useRef, useState } from 'react'
+import { memo, useCallback, useEffect, useRef, useState } from 'react'
```

Comment:
```suggestion
import { useCallback, useEffect, useRef, useState } from 'react'
```
memo는 import만 하고 사용은 안한 것 같아요!